earth_enterprise/src/fusion/autoingest/FromStorageGen.pl

```diff
@@ -67,25 +67,20 @@ ()
 
```

Comment:
Several lines above this, the line `print $fh "#include <khRefCounter.h>\n\n";` should be removed.